### PR TITLE
LOG-2720: Add clusterID to data model for vector

### PR DESCRIPTION
**Pull request by jcantrill**
### Description
This PR:
* add support to vector to include openshift.cluster_id in the log payload
* Enables some audit functional testing for vector

### Links
* https://issues.redhat.com/browse/LOG-2720

### CODE REVIEW

File: internal/generator/vector/normalize.go [hunk 1]
```diff
@@ -11,7 +11,8 @@ import (
 )
 
 const (
-	FixLogLevel = `
+	ClusterIDFormat = `.openshift.cluster_id = "%s"`
```
Comment (vimalk78) [inline internal/generator/vector/normalize.go:14]: Can we generate this line based on an env variable? 
```
	ClusterIDFormat = `.openshift.cluster_id = "${OPENSHIFT-CLUSTER-ID:-abc-123-mock-clusterid}"`
```
this way code changes are minimal, dont have to pass around clusterID in the generator functions. Also makes generated conf independent of a specific cluster.

Comment (jcantrill) [reply internal/generator/vector/normalize.go]: Updated to use env var

File: test/helpers/types/types.go [hunk 2]
```diff
@@ -179,6 +180,7 @@ type LinuxAuditLog struct {
 	OpenshiftLabels  OpenshiftMeta    `json:"openshift"`
 	Timing           `json:",inline"`
 	Level            string `json:"level,omitempty"`
+	WriteIndex       string `json:"write_index,omitempty"`
```
Comment (vimalk78) [inline test/helpers/types/types.go:183]: we dont need `write_index` now. it has been removed in a recent PR

**Pull request merged**